src/modules/AllWidgets.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace WPML\PB\Elementor\Modules;
+
+
+class AllWidgets extends \WPML_Elementor_Media_Node {
+
+	/**
+	 * @param array $settings
+	 * @param string $target_lang
+	 * @param string $source_lang
+	 *
+	 * @return array
+	 */
+	public function translate( $settings, $target_lang, $source_lang ) {
```

Comment:
Please use camelCase when possible for this new class.

Reply:
fixed.